packages/edit-site/src/components/header/document-actions/style.scss

```diff
@@ -7,14 +7,31 @@
 	position: relative;
 
 	.edit-site-document-actions__title-wrapper {
+		display: flex;
+		flex-direction: row;
+		justify-content: center;
+		align-items: center;
+		transition: transform 0.2s;
+		@include reduce-motion(transition);
```

Comment:
cc @jeyip I had to move the animation to this wrapper element -- let me know if this looks good!

Reply:
Looks good to me! I'll spin up the PR quickly to double-check.

Reply:
Yep. It looks like we're good to go. 🙂